pkg/connector/roles.go

```diff
@@ -52,21 +52,23 @@ func removeRolesPrefix(role string) string {
 }
 
 func (o *roleBuilder) List(ctx context.Context, parentResourceID *v2.ResourceId, pToken *pagination.Token) ([]*v2.Resource, string, annotations.Annotations, error) {
+	var resources []*v2.Resource
 	policy, err := o.projectsClient.GetIamPolicy(ctx, &iampb.GetIamPolicyRequest{
 		Resource: fmt.Sprintf("projects/%s", o.bigQueryClient.Project()),
 	})
-	if err != nil {
-		return nil, "", nil, wrapError(err, "failed to get IAM policy")
+	if !isPermissionDenied(ctx, err) {
+		return nil, "", nil, wrapError(err, "listing roles failed")
```

Comment:
I think this is getting an IAM policy, not a role?

Reply:
Fixed